### PR TITLE
Move jQuery plugins into modules and add them to the bundle

### PR DESCRIPTION
**Pull request by ChristophWurst**
This organizes our jquery code a lot and also moves more code to the bundle.

### CODE REVIEW

**Comment by ChristophWurst** [comment]
@skjnldsv this jquery code is so ugly it will make you cry again

File: core/src/jquery/tipsy.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import $ from 'jquery'
+
+/**
+ * $ tipsy shim for the bootstrap tooltip
+ * @deprecated
+ */
+$.fn.tipsy = function (argument) {
```
Comment (skjnldsv) [inline core/src/jquery/tipsy.js]: do we use it anywhere?? :scream:

Comment (ChristophWurst) [reply core/src/jquery/tipsy.js]: I hope we don't because this has been deprecated for ages. I'm fine with removing it finally in a follow-up PR!

Comment (rullzer) [reply core/src/jquery/tipsy.js]: Move to bundle => deprecate properly in the dev announcements => kill in 17

Comment (MorrisJobke) [reply core/src/jquery/tipsy.js]: Those should be all locations:

bildschirmfoto 2019-01-29 um 14 10 34

**Review by MorrisJobke — APPROVED** [review]

**Comment by ChristophWurst** [comment]
CI failure is unreated

**Review by rullzer — APPROVED** [review]
Gave it a quick tests. Seems to work :rocket:

**Pull request merged**